docs/hub/repositories-pull-requests-discussions.md

```diff
@@ -59,6 +98,17 @@ You can also hide a comment. Hiding a comment is irreversible, and nobody will b
 
 Read also [moderation](./moderation) to see how to report an abusive comment.
 
+## Disabling Discussions / Pull Requests
```

Comment:
Maye this part should be in `repositories-settings.md`, wdyt?

Reply:
Agree! (and still link from here somewhere)